### PR TITLE
RHDEVDOCS-6183: Content creation for EnableRolloutsUI field

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.15 [gitops-docs-1.15]

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6183

Link to docs preview:

[Enabling Argo Rollouts UI on a default Argo CD instance](https://85537--ocpdocs-pr.netlify.app/openshift-gitops/latest/argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.html#gitops-enabling-argo-rollouts-ui-on-a-default-argo-cd-instance_using-argo-rollouts-for-progressive-deployment-delivery)

QE review:
- [x] QE has approved this change.

SME review: ansingh@redhat.com
QE review: @varshab1210 
Peer review: @adellape 

Additional information:

### CODE REVIEW

**Review by anandrkskd — COMMENTED** [review]
Just one nitpick, otherwise LGTM!

**Comment by anandrkskd** [comment]
lgtm

File: argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc [hunk 1]
```diff
@@ -26,6 +26,9 @@ include::modules/gitops-installing-argo-rollouts-cli-on-linux.adoc[leveloffset=+
 // Installing Argo Rollouts CLI on Mac OS
 include::modules/gitops-installing-argo-rollouts-cli-on-mac-os.adoc[leveloffset=+1]
 
+// Enabling Argo Rollouts UI on a default Argo CD instance
+include::modules/gitops-enabling-argo-rollouts-ui-on-a-default-argo-cd-instance.adoc[leveloffset=+1]
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc]: Fixed.

**Comment by varshab1210** [comment]
LGTM, thanks

File: modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+----
+<1> Set this value to `true` to configure the `enableRolloutsUI` field.
+
+. Click *Save*.
+
+. In the *Administrator* perspective of the web console, navigate to the {rh-app-icon} menu -> *OpenShift GitOps* -> *Cluster Argo CD*. The login page of the Argo CD Web UI is displayed in a new window.
+
+. To access the Argo Rollouts UI in the Argo CD Web UI, configure a sample application that includes the Argo Rollouts resources.
+
++
```
Comment (adellape) [inline modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc:50]: Remove blank line here:
```suggestion
. To access the Argo Rollouts UI in the Argo CD Web UI, configure a sample application that includes the Argo Rollouts resources.
+
```

File: modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+To enable Argo Rollouts UI on an Argo CD instance, complete the following steps.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+
+* You have installed the {gitops-title} Operator on your {OCP} cluster.
+
+* You have configured the xref:../argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.html#gitops-creating-rolloutmanager-custom-resource_using-argo-rollouts-for-progressive-deployment-delivery[RolloutManager] CR.
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc:17]: Fixed.

[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc:17]: Fixed.

File: modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+. Log in to the {OCP} web console.
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
+
+. Select the Argo CD instance in the *Argo CD* tab under the `openshift-gitops` namespace.
+
+. Click *YAML* and add the following configuration to configure the Argo Rollouts UI.
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc:29]: Fixed.

File: modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
++
+.Example enable Argo Rollouts UI in the Argo CD Custom Resource (CR)
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc:31]: Fixed.

File: modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc (hunk 2 above)
Comment (Dhruv-Soni11) [reply modules/gitops-enabling-argo-rollouts-ui-on-an-argo-cd-instance.adoc:50]: Fixed.

**Review by sheriff-rh — APPROVED** [review]
Changes LGTM; merging.

**Pull request merged**